### DISCUSSION TITLE
Windows 11 Remove Widgets

### DISCUSSION THREAD

**Discussion by canyon1991** [farag2/Sophia-Script-for-Windows#293] [Q&A] [answered]
I am not 100% this feature is available or not but I don't recall seeing it a couple of weeks ago when I used the script.

Option to remove(uninstall) Widgets on Windows 11. It uses resources and most people will not use it. 

Guide below copied from https://www.ghacks.net/2021/10/08/how-to-uninstall-widgets-in-windows-11/

"1. Open a Command Prompt (or PowerShell) as an Administrator.

2. Type the following command in the window, and hit the Enter key

winget uninstall "windows web experience pack"

You will need to agree to the Microsoft's Store's terms and conditions. Type Y to accept, or N to cancel the operation.

If you chose Yes, the Winget package manager will delete the Widgets' app's package which has the name: MicrosoftWindows.Client.WebExperience. You will see a message in the command window that says Successfully Uninstalled. Check the Task Manager and you should no longer see the Widgets among the list of background processes. The toggle for the app will also be removed from the Taskbar settings. "

**Comment by farag2** [comment, marked as answer]
I already provide a feature to uninstall this UWP app: just run the appropriate function.

**Comment by canyon1991** [comment]
Thank you I will have to re-run the script again and have look then as I didn't see it last time (about 4 weeks ago).

**Comment by farag2** [comment]
It won't be named like Widgets. I cannot recall the actual name, but if you don't know it, you won't guess it.

**Comment by farag2** [comment]
https://techdows.com/2021/11/uninstall-widgets-windows-11.html

I was right. :)
